tests/unit/AgmExtension.spec.ts

```diff
@@ -94,27 +97,35 @@ describe('AGM Extension view', () => {
 
   it('displays expandable help properly', async () => {
     expect(wrapper.find('.help-label').text()).toContain('Help with')
+
+    // verify help section is not visible
     expect(wrapper.find('.help-section').isVisible()).toBe(false)
     await wrapper.find('.help-btn').trigger('click')
-    await flushPromises()
+
+    // verify help section is now visible
     expect(wrapper.find('.help-section').isVisible()).toBe(true)
+
+    // verify title and text
     expect(wrapper.find('.agm-extension-help > h3').text()).toBe('AGM Extension Help')
     expect(wrapper.find('.agm-extension-help > div').text()).toContain('A company must have')
     expect(wrapper.find('.agm-extension-help > div').text()).toContain('Shareholders entitled')
     expect(wrapper.find('.agm-extension-help > div').text()).toContain('If a company does not')
   })
 
-  it('show validation errors when Extension Request is incomplete', async () => {
+  it('sets fee summary data properly', () => {
+    expect(wrapper.vm.filingData).toEqual([{ filingTypeCode: 'AGMDT', entityType: 'BEN' }])
```

Comment:
When SETTING data in a unit test, we should use the enums (eg, see line 40).

When GETTING data in a unit test, it's better to check the expected string value. This improves the chance of catching enum issues.